packages/styles/src/themes/pentahoPlus.ts

```diff
@@ -1098,8 +1132,13 @@ const pentahoPlus = makeTheme((theme) => ({
             borderColor: theme.colors.textDimmed,
           },
         },
-        panel: {
-          borderColor: theme.colors.textDimmed,
+        panelOpenedDown: {
+          top: 0,
+          ...popperStyles(theme),
```

Comment:
![image](https://github.com/user-attachments/assets/ab5c6506-0fd6-4ec3-881c-c797d99a5396)

If this is the spec, then the spec must be wrong... 😅
there's a huge mismatch between the popper and the input "header"

looking at the "current" TimePicker spec, this "popper" shouldn't be used:

![image](https://github.com/user-attachments/assets/da659b26-3e5b-494c-bfc7-50f21759562d)

We need to better understand where exactly this new popper is to be used first